airbyte-ci/connectors/pipelines/pipelines/cli/auto_update.py

```diff
@@ -0,0 +1,122 @@
+# HELPERS
+
+import importlib
+import logging
+import os
+import sys
+
+import asyncclick as click
+import requests
+from pipelines import main_logger
+from pipelines.consts import LOCAL_PIPELINE_PACKAGE_PATH
+
+__installed_version__ = importlib.metadata.version("pipelines")
+
+BINARY_UPGRADE_COMMAND = "make tools.airbyte-ci.install"
+DEV_UPGRADE_COMMAND = "make tools.airbyte-ci-dev.install"
```

Comment:
Can we make these upgrade command download and execute the shell scripts from `https://raw.github.com`? The upgrade would otherwise be tied to the existence of the repo.

Reply:
You know what yes, me wanting to tie it to the makefile was wrong! Will change